src/lib/__tests__/Preview-test.js

```diff
@@ -1943,6 +1948,108 @@ describe('lib/Preview', () => {
         });
     });
 
+    describe('createLog()', () => {
+        it('should create a log object containing correct file info properties', () => {
+            const id = '12345';
+            preview.file = {
+                id
+            };
+
+            const log = preview.createLog();
+            expect(log.timestamp).to.exist;
+            expect(log.file_id).to.equal(id);
+            expect(log.file_version_id).to.exist;
+            expect(log.content_type).to.exsit;
```

Comment:
typeo?

Reply:
I want to know why these tests passed, locally, if this was here :O